damus/Views/Muting/MutelistView.swift

```diff
@@ -99,13 +82,30 @@ struct MutelistView: View {
                                     RemoveAction(item: .thread(note_id, nil))
                                 }
                         } else {
-                            Text("Error retrieving muted event", comment: "Text for an item that application failed to retrieve the muted event for.")
+                            Text("Error retrieving muted event")
                         }
                     }
                 }
             }
+            Section(
+                header: Text("Users"),
```

Comment:
Related to my other comment. This line can be:
```suggestion
                header: Text("Users", comment: "Section header title for a list of muted users."),
```